_alerts/2017-07-12-potential-split.md

```diff
@@ -0,0 +1,105 @@
+---
+## This file is licensed under the MIT License (MIT) available on
+## http://opensource.org/licenses/MIT.
+
+title: "Potential disruptive chain split"
+shorturl: "potential-split"
+active: true
+banner: "Warning: Bitcoin may be unsafe to use starting July 31st (click here for details)"
+
+## 7/12 to 7/28: "warning" (orange); 7/29 until resolution: "alert" (red)
+bannerclass: "warning"
+---
+{% assign start='<span class="date">2017/08/01 00:00 UTC</span>' %}
+
+*Last updated: <span class="date">2017/07/12 12:00 UTC</span>.  This
+page will be updated when new information becomes available.  See the
+[list of updates][].*
+
+At {{start}}, Bitcoin confirmation scores may become unreliable for
+an unknown length of time.  This means that any bitcoins you receive
+after that time may later disappear from your wallet or be a
+type of bitcoin that other people will not accept as payment.
```

Comment:
We should also add a sentence to the effect of:

`Leading up to, during and after this period, bitcoin may also experience significant price fluctuations in relation to other currencies.`